lib/constable/services/hub_profile_provider.ex

```diff
@@ -1,24 +1,20 @@
 defmodule Constable.Services.HubProfileProvider do
-  alias ConstableWeb.Endpoint
   alias Constable.Services.ProfileProvider
   @behaviour ProfileProvider
-  use Memoize
-
-  @default_image_path "/images/ralph.png"
 
   @impl ProfileProvider
-  defmemo profile_url(user) do
+  def fetch_profile_url(user) do
     "#{Application.fetch_env!(:constable, :hub_url)}/people/#{fetch_slug(user)}"
   end
 
   @impl ProfileProvider
-  defmemo image_url(user) do
+  def fetch_image_url(user) do
```

Comment:
I updated these to be `fetch_` to disambiguate from the attributes in the user (e.g. `user.profile_url`).